### DISCUSSION TITLE
Add dual account support?

### DISCUSSION THREAD

**Discussion by TiPa7** [community/community#37385] [Mobile]
As far as I see the features of the app, you can only use one private account. I think it would help if you could use multiple private ones.

**Comment by 3kh0** [comment]
Why do you need more than one?

**Reply by drewcassidy** [reply]
I have an account for work and an account for personal projects.

**Comment by tawfik-s** [comment]
1- do all your contributions to a single account to be noticed if you have two accounts you will split your effort and will not get noticed
2- create only one and put all your effort into it then you will have a great account with great contributions

**Reply by tawfik-s** [reply]
if it helps you mark as an answer @TiPa7

**Reply by airtower-luna** [reply]
One more important aspect to note: Having more than one *free* account is [against the GitHub ToS](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#b-account-terms).

**Reply by TiPa7** [reply]
> One more important aspect to note: Having more than one *free* account is [against the GitHub ToS](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#b-account-terms).

Well that's good to know. I thought it might be a thing because the app has the feature of adding another enterprise account.
Thanks

**Comment by DenverCoder1** [comment]
An Organization might be what you're looking for, as that allows using more than one name logged in simultaneously.

For example, I use @DenverCoderOne for forks and open source contributions and @DenverCoder1 for personal projects.

There's no need to log out to switch between working on both.

You can create an organization while logged into your main account and that allows you to separate your repos between them:

![image](https://user-images.githubusercontent.com/20955511/198154917-ff3d548f-ac0a-4920-9106-b983b824425f.png)

Note: There are some differences between user and organization accounts. For example, when contributing to an open source project with an organization account, you can't allow project maintainers to push to your branch. On the other hand, repos owned by organization accounts allow for more granular access permissions.

For most cases, I'd recommend just keeping all your repos in one place since that's the simplest and what most people do, plus it makes your profile look better if you're interested in being recruited. But for cases where you don't want certain repos to appear on your main account or to be organized separately, organizations are the best option (at least for free accounts).

**Comment by aconsuegra** [comment]
Hi @TiPa7, thanks for your feature request! We are right now investigating what it would take to support several personal accounts in the GitHub Mobile apps. We don't have a timeline yet, but we are looking into it!

**Comment by jsoref** [comment]
It'd be helpful if the various requests were all linked together. I'm currently using https://github.com/community/community/discussions/39693#discussioncomment-5754343 for that purpose.

**Comment by jsoref** [comment]
This has been added as noted in https://github.com/orgs/community/discussions/39693#discussioncomment-6742948
